ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -35,14 +38,21 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
   }
 
   /**
-   * Loads all targets and the number of targets under current scope.
+   * Loads queried targets, the number of targets under current scope, and
+   * projects for filtering options.
    *
-   * @returns {Promise<{totalItems: number, targets: [TargetModel]}>}
+   * @returns {Promise<{totalItems: number, targets: [TargetModel], projects: [ScopeModel]}>}
    */
-  async model({ search = '' }) {
+  async model({ search = '', scopes }) {
     // TODO: Filter targets by scope we're in manually
```

Comment:
Will this be a separate PR?

Reply:
Yes that was the plan. I wanted to tackle this after getting the scope filter to work.